src/array.rs

```diff
@@ -825,7 +909,9 @@ impl<T: Element, D: Dimension> PyArray<T, D> {
     ///
     /// # Panics
     ///
-    /// Panics if the allocation backing the array is currently borrowed.
+    /// Panics if the allocation backing the array is currently borrowed or
+    /// if the array is not writeable.
```

Comment:
not writable in terms of [flags](https://numpy.org/doc/stable/reference/generated/numpy.ndarray.flags.html), right?

Linking to the corresponding numpy doc might help.

Reply:
Indeed, but considering the documentation the flag is really meant to mark arrays backed by data that cannot be written (in contrast to should not), e.g. read-only memory maps which will cause an access violation if written to.

In any case, I added the link to the flags documentation.